apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,69 @@
 # API Changelog
 
+## BFD-1923: Add Org Name Display to Organizations in V2
+Add the display name for NPI Organizations to the contained resource for organizations.
+
+Old Organization Resource Mapping Display Name in V2:
+```json
+      "contained" : [ {
+        "resourceType" : "Organization",
+        "id" : "provider-org",
+        "meta" : {
+          "profile" : [ "http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-Organization" ]
+        },
+        "identifier" : [ {
+          "type" : {
+            "coding" : [ {
+              "system" : "http://terminology.hl7.org/CodeSystem/v2-0203",
+              "code" : "PRN"
+            } ]
+          },
+          "value" : "999999"
+        }, {
+          "type" : {
+            "coding" : [ {
+              "system" : "http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBIdentifierType",
+              "code" : "npi"
+            } ]
+          },
+          "system" : "http://hl7.org/fhir/sid/us-npi",
+          "value" : "0000000000"
+        } ],
+        "active" : true,
+      } ],
+```
+
+New Mapping in V2:
+```json
+      "contained" : [ {
+        "resourceType" : "Organization",
+        "id" : "provider-org",
+        "meta" : {
+          "profile" : [ "http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-Organization" ]
+        },
+        "identifier" : [ {
+          "type" : {
+            "coding" : [ {
+              "system" : "http://terminology.hl7.org/CodeSystem/v2-0203",
+              "code" : "PRN"
+            } ]
+          },
+          "value" : "999999"
+        }, {
+          "type" : {
+            "coding" : [ {
+              "system" : "http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBIdentifierType",
+              "code" : "npi"
+            } ]
+          },
+          "system" : "http://hl7.org/fhir/sid/us-npi",
+          "value" : "0000000000"
+        } ],
+        "active" : true,
+        "name" : "Fake ORG Name"
+      }
+```
+
 ## BFD-2145: Removal of duplicated careteam member entries and their extensions.
 Corrected an issue where duplicate entries for a provider were being supplied in the Explanation of Benefit careTeam element, making it cumbersome to process and link care team members to items.
```

Comment:
Why was this deleted?

Reply:
It's not. I _think_ the json/jsonc codeblock(s) is complaining about the json syntax and isn't closing as a result. The red highlighting, normally depicting removal, seems to be a githubism.